docs/api/covidcast-signals/hhs.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Department of Health & Human Services
+parent: Data Sources and Signals
+grand_parent: COVIDcast Epidata API
+---
+
+# SOURCE NAME
+{: .no_toc}
+
+* **Source name:** `hhs`
+* **First issued:** TBD
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, hrr, msa, state (see [geography coding docs](../covidcast_geography.md))
```

Comment:
```suggestion
* **Available for:** state (see [geography coding docs](../covidcast_geography.md))
```

Reply:
```suggestion
* **Available for:** state, national (see [geography coding docs](../covidcast_geography.md))
```

Reply:
does the indicator currently do nation? We add hhs and nation if we wanted

Reply:
not yet, but I think we should add them before the release.

it looks like GeoMapper only supports [fips,zip]->nation, would you take a look at how hard it would be to add a state->nation crosswalk?

Reply:
think it should be pretty simple, ill work on it